packages/mdc-chips/_mixins.scss

```diff
@@ -438,6 +465,15 @@ $mdc-chip-ripple-target: ".mdc-chip__ripple";
     padding-right: $padding;
     padding-left: $padding;
   }
+
+  $margin: 0 - $padding;
+
```

Comment:
nit: -1 newline

Reply:
This whole chunk should not have been there. Removed from CL, but forgot to update PR.